pkg/server/commands/listusers/list_users_rpc_test.go

```diff
@@ -2060,6 +2060,7 @@ func TestListUsersCycleDetection(t *testing.T) {
 					}},
 				},
 				visitedUsersetsMap: visitedUsersets,
+				Depth:              25,
```

Comment:
nit (non-block): We manually define 25 for Depth in various places. It would be nice if we had this as a `const` somewhere appropriate.